dbms/src/Storages/Page/WriteBatchWrapperImpl.h

```diff
@@ -66,7 +66,7 @@ class WriteBatchWrapper : private boost::noncopyable
     }
 
 #ifdef DBMS_PUBLIC_GTEST
-    WriteBatchWrapper(WriteBatch && wb_)
+    WriteBatchWrapper(WriteBatch && wb_) // NOLINT(google-explicit-constructor), for gtest
```

Comment:
there are many statements like

```cpp
page_writer->write(std::move(batch), nullptr);
```

which will implicitly call the move constructor. If we add explicit here, then we need to rewrite it to 

```cpp
page_writer->write(WriteBatchWrapper(std::move(batch)), nullptr);
```

Since it is only used in gtests, so keep it.